x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/controls.tsx

```diff
@@ -267,7 +270,7 @@ export const Controls: FC<Props> = React.memo(
         <EuiFlyout
           ownFocus
           size="m"
-          onClose={() => setShowFlyout(false)}
+          onClose={() => deselect()}
```

Comment:
nit: 
```suggestion
          onClose={deselect}
```

Reply:
Good one - updated in fe53999a7140c36ea41e3524c98ffa8fa5db42d3